plugins/linuxv2/model/namespace/namespace.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package namespace;
+
+message Namespace {              /* Linux network namespace */
+    enum NamespaceType {
```

Comment:
`namespace.Namespace_NamespaceType` does not look very clean.

I would suggest to keep just `Type` for the enum.